### PR TITLE
community: learn: add reason-react with apollo

### PR DESCRIPTION
**Pull request by wawhal**
### Description

Adds ReasonReact with Apollo tutorial
### Affected components 

- Community Content

### CODE REVIEW

File: community/learn/graphql-tutorials/remote-hasura.yaml.sample [hunk 1]
```diff
@@ -59,6 +59,16 @@ spec:
       name: android-apollo
       path: ./community/learn/graphql-tutorials/tutorials/android-apollo/tutorial-site
     name: android-apollo
+  - containers:
```
Comment (shahidhk) [inline community/learn/graphql-tutorials/remote-hasura.yaml.sample]: What is `elm-apollo`?

File: community/learn/graphql-tutorials/tutorials/react-native-apollo/tutorial-site/content/update-delete-mutations.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Update mutations and automatic cache updates"
-metaTitle: "Optimistic UI updates after mutations | GraphQL React Native Apollo Tutorial"
-metaDescription: "We will use the Apollo Client Optimistic Response to perform UI updates after a GraphQL mutation in the React Native app"
+metaTitle: "Automatic cache updates after mutations | GraphQL React Native Apollo Tutorial"
```
Comment (shahidhk) [inline community/learn/graphql-tutorials/tutorials/react-native-apollo/tutorial-site/content/update-delete-mutations.md]: Is this relevant to this PR?

Comment (wawhal) [reply community/learn/graphql-tutorials/tutorials/react-native-apollo/tutorial-site/content/update-delete-mutations.md]: Not really. But it's a single line bug. We can merge it.

File: community/learn/graphql-tutorials/remote-hasura.yaml.sample (hunk 1 above)
Comment (wawhal) [reply community/learn/graphql-tutorials/remote-hasura.yaml.sample]: Fixed. Resolve.

File: community/learn/graphql-tutorials/tutorials/react-native-apollo/tutorial-site/content/update-delete-mutations.md (hunk 2 above)
Comment (shahidhk) [reply community/learn/graphql-tutorials/tutorials/react-native-apollo/tutorial-site/content/update-delete-mutations.md]: Okay.

**Comment by idkjs** [comment]
Reason Tutorial final app doesnt compile. Error is here: https://github.com/hasura/graphql-engine/blob/dde990865096fb2123fa27675355dfe661ab9b69/community/learn/graphql-tutorials/tutorials/reason-react-apollo/app-final/src/todo/TodoPublicList.re#L63

```ocaml
  We've found a bug for you!
  /Users/prisc_000/Downloads/app-final/src/todo/TodoPublicList.re 63:22

  61 ┆   "variables": fetchOlderTodosQuery##variables
  62 ┆ };
  63 ┆ let apolloData = c##query(query);
  64 ┆ apolloData
  65 ┆ |> Js.Promise.then_(gqlResp => {

  The value c can't be found
```

I am not following what you were trying to do so can't fix it. 

Also, its really hard to contribute on here because of your mono repo structure. Do you have a suggested way of contributing on issues on different parts of the repo?
